src/app/portlets/content-types/fields/service/field-drag-drop.service.ts

```diff
@@ -181,16 +181,18 @@ export class FieldDragDropService {
         return this.isDraggingFromSource(source);
     }
 
+    // tslint:disable-next-line:cyclomatic-complexity
     private shouldMoveRow(
         _el: HTMLElement,
         source: HTMLElement,
         handle: HTMLElement,
         _sibling: HTMLElement
     ): boolean {
+        const noDrag = !handle.classList.contains('no-drag');
```

Comment:
Did you try to preventDefault or stopProppagation?

Reply:
it does not work